frontend/src/ErrorPage.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  root: {
+    margin: 'auto',
+    width: '100%',
+    maxWidth: 800,
+    paddingTop: 100,
+    textAlign: 'center',
+  },
+});
+
+export default function ErrorPage() {
```

Comment:
does this page show up any time user tries to visit an url for an alert that does not exist?

Reply:
yup!